src/main/java/ru/skypro/homework/service/CommentService.java

```diff
@@ -0,0 +1,101 @@
+package ru.skypro.homework.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.HttpClientErrorException;
+import ru.skypro.homework.dto.Comment;
+import ru.skypro.homework.dto.Comments;
+import ru.skypro.homework.dto.CreateOrUpdateComment;
+import ru.skypro.homework.dto.Role;
+import ru.skypro.homework.entity.AdComment;
+import ru.skypro.homework.entity.User;
+import ru.skypro.homework.exception.UserNotFoundException;
+import ru.skypro.homework.mapper.CommentMapper;
+import ru.skypro.homework.repository.CommentRepository;
+import ru.skypro.homework.repository.UserRepository;
+
+import javax.persistence.EntityNotFoundException;
+import java.time.LocalDateTime;
+
+@Service
+public class CommentService {
+
+    private static final Logger logger = LoggerFactory.getLogger(CommentService.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(CommentService.class); 
```
потому что static И final